.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+* @vtex-apps/store-framework-devs
+* @vtex-apps/checkout-ui
```

they actually need to be in the same line, else only `@vtex-apps/store-framework-devs` or `@vtex-apps/checkout-ui` will be asked for review
```suggestion
* @vtex-apps/store-framework-devs @vtex-apps/checkout-ui
```